docs/release-notes/release-notes-0.18.3.md

```diff
@@ -33,6 +33,9 @@
   created](https://github.com/lightningnetwork/lnd/pull/8921) when multiple RPC
   calls are made concurrently.
 
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/8174) old payments that
+  are stuck inflight. Check more info [here](https://github.com/lightningnetwork/lnd/issues/8146).
+
```

Comment:
wondering whether the three commits on top of this PR are related ?

Reply:
you mean the last three? yeah they are related as they are required to get the itest pass.